tests/integration-tests/tests/ultraserver/test_gb200/test_gb200/check_imex_status.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+
+# Maximum number of attempts
+MAX_ATTEMPTS=10
+
+# Function to run the command and check status
+verify_imex_is_up() {
+    local command="/usr/bin/nvidia-imex-ctl -N -j"
+    local file_prefix="result"
+
+    for ((i=1; i<=MAX_ATTEMPTS; i++)); do
+        echo "Attempt $i for IMEX"
+
+        command_output=$(${command} 2>/dev/null)
+        # Check if the status is UP
+        if echo "$command_output" | grep -q " \"status\": \"UP\""; then
```

Comment:
Why not using jq to retrieve the specific field?
The risk of using grep is that you match a "status' field that is not the right one.